src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -1019,6 +1019,9 @@ zocl_xclbin_cus_support_intr(struct drm_zocl_dev *zdev)
 
 	for (i = 0; i < zdev->ip->m_count; ++i) {
 		ip = &zdev->ip->m_ip_data[i];
+		if(xclbin_protocol(ip->properties) & 0x2) {
```

Comment:
Suggest to add a space after "if", replace & with "==" and use AP_CTRL_NONE instead of 2
if (xclbin_protocol(ip->properties) == AP_CTRL_NONE) {

Reply:
fixed as per comments